vowpalwabbit/gen_cs_example.h

```diff
@@ -166,16 +166,16 @@ void gen_cs_example(cb_to_cs& c, example& ec, CB::label& ld, COST_SENSITIVE::lab
   }
 }
 
-void gen_cs_test_example(v_array<example*> examples, COST_SENSITIVE::label& cs_labels);
+void gen_cs_test_example(multi_ex examples, COST_SENSITIVE::label& cs_labels);
```

Comment:
Should these by pass by reference instead?  Or maybe const?

Reply:
Changed to pass by reference